tests/test_other.py

```diff
@@ -199,6 +199,15 @@ def test_emcc_output_worker_mjs(self):
     with open('hello_world.worker.js') as f:
       self.assertContained('import(', f.read())
 
+  def test_export_es6_implies_modularize(self):
+    self.run_process([EMCC, path_from_root('tests', 'hello_world.c'), '-s', 'EXPORT_ES6=1', '-s', 'ASSERTIONS=0'])
+    src = open('a.out.js').read()
+    self.assertContained('export default Module;', src)
+
+  def test_export_es6_requires_modularize(self):
+    err = self.expect_fail([EMCC, path_from_root('tests', 'hello_world.c'), '-s', 'EXPORT_ES6=1', '-s', 'MODULARIZE=0', '-s', 'ASSERTIONS=0'])
```

Comment:
and here?

Reply:
Good catch, those are not needed. Have updated the PR. Thanks!